spec/mercure.md

```diff
@@ -213,20 +76,29 @@ superior to leverage multiplexing and other performance-oriented related feature
 versions.
 
 The subscriber specifies the list of topics to get updates from by using one or several query
-parameters named `topic`. The `topic` query parameters **MUST** contain topic selectors. See
-(#topic-selectors).
+parameters named `match`. The subscriber only receives updates for the topics exactly matching
+one of the `match` query parameters.
 
-The protocol doesn't specify the maximum number of `topic` parameters that can be sent, but the hub
-**MAY** apply an arbitrary limit. A subscription is created for every provided `topic` parameter.
-See (#subscription-events).
+In addition to `match` query parameters, the subscriber can pass other topic matchers by passing
+query parameters starting by the string `match` and followed by the topic matcher type.
+
+The subscriber receives updates for all topics matching at least a topic matcher according to
+the matcher type rules.
+
+The hub **SHOULD** support the `Regexp`, `URLPattern` and `URITemplate` matcher types.
+The corresponding query parameters are respectively `matchRegexp`, `matchURLPattern` and `matchURITemplate`.
```

Comment:
```suggestion
The corresponding query parameters are respectively `matchRegExp`, `matchURLPattern` and `matchURITemplate`.
```

Reply:
The RFC use the `Regexp` case, let's be consistent with it: https://www.ietf.org/rfc/rfc9485.html